docs/source/models.rst

```diff
@@ -86,6 +86,17 @@ You can use the following transform to normalize::
 An example of such normalization can be found in the imagenet example
 `here <https://github.com/pytorch/examples/blob/42e5b996718797e45c46a25c55b031e6768f8440/imagenet/main.py#L89-L101>`_
 
+The values of `mean` and `std` were obtained by running::
+
+    mean = mean([mean(transform(img)) for img in subset(dataset)])
+    std = mean([std(transform(img)) for img in subset(dataset)])
```

Comment:
I'm not sure if this will be clear to everyone. Its a little bit confusing that `mean` is value and function at the same time. As it stands this would not run if everything is defined. Ideas?

Reply:
yeah, this is a bit confusing.

Maybe something like
```python
dataset = datasets.ImageNet('.', split='train')
transform = T.Compose([T.Resize(256), T.CenterCrop(224])

mean = np.mean([np.mean(transform(img)) for img in subset(dataset)])
std = np.mean([np.std(transform(img)) for img in subset(dataset)])
```

Reply:
1. In that case I would also include the imports to make it complete.
2. I don't think `np.mean` and `np.std` work on `PIL` images or lists. If we go for a complete example I suggest we add `ToTensor()` to the transform and use the corresponding `torch` functions.

Reply:
yeah, need to add `ToTensor` as well.
about using torch functions, it could be, but they don't support lists, so need to convert the list to a tensor first. your call

Reply:
I'll give it a shot and get back to you for review